.circleci/config.yml

```diff
@@ -168,6 +168,24 @@ commands:
           working_directory: << parameters.directory >>
           when: always
 
+  scan:
```

Comment:
Needed this because the other jobs also `archive`, which we don't ned for the new job.